src/index.html

```diff
@@ -1,13 +1,60 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" >
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Open+Sans:ital,wght@0,300;0,400;0,500;0,600;0,700;0,800;1,400&family=Roboto:wght@400;500;700&family=Space+Grotesk:wght@700&family=Space+Mono&family=Tinos:wght@400;700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="page_body">
+    <div class="cards" data-qa="card">
+      <img
+        src="./images/imac.jpeg"
+        alt="APPLE A1419 iMac 27&#34; Retina 5K Monoblock (MNED2UA/A)"
+        class="cards-img"
+      >
+
+      <div class="cards-text">
```

Comment:
Follow the BEM, please. The text should be an element of the card. Check the BEM structure according to the [checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md) everywhere 
```suggestion
      <div class="cards__text">
```